spec/acceptance/broker_spec.rb

```diff
@@ -91,7 +91,7 @@ class { 'kafka::broker':
 
       describe file('/opt/kafka/config/server.properties') do
         it { is_expected.to be_file }
-        it { is_expected.to be_owned_by 'kafka' }
+        it { is_expected.to be_owned_by 'root' }
```

Comment:
is it on purpose that the file is owned by root and not kafka?

Reply:
@bastelfreak i either fix the test or the manifest, what do you prefer?

Reply:
okay adjusted the tests may be the better idea, otherwise we may cause breaking changes.